### PR TITLE
merge parkinsonism-dystonia, infantile, 2

### LINKED ISSUE #6933: MONDO:0018130 & MONDO:0054836 [Merge]
**Mondo term (ID and Label)**
MONDO:0054836 parkinsonism-dystonia, infantile, 2

**Reason for deprecation**
Appears to be same disease as MONDO:0018130 
-Orphanet reports the OrphaID is an exact match to the OMIM number http://www.orpha.net/ORDO/Orphanet_352649
-OMIM points to Orphanet and now also uses the Mondo pref name (and Orpha's pref name) as synonyms on their record https://www.omim.org/entry/618049

**Term to be merged with**
MONDO:0018130 brain dopamine-serotonin vesicular transport disease
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6933

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -391605,19 +391605,26 @@ is_a: MONDO:0020044 {source="Orphanet:352641"} ! autosomal recessive metabolic c
...
 id: MONDO:0018130
 name: brain dopamine-serotonin vesicular transport disease
-def: "Brain dopamine-serotonin vesicular transport disease is a newly discovered infantile-onset neurometabolic disease characterized by dystonia, parkinsonism, nonambulation, autonomic dysfunction, developmental delay and mood disturbances." [Orphanet:352649]
+def: "An infantile-onset neurometabolic disease characterized by dystonia, parkinsonism, nonambulation, autonomic dysfunction, developmental delay and mood disturbances." [https://orcid.org/0000-0001-5208-3432, Orphanet:352649]
 subset: gard_rare {source="GARD:13594"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_disease {source="Orphanet:352649"}
 subset: orphanet_rare {source="Orphanet:352649"}
 subset: rare
+synonym: "Parkinsonism-dystonia, infantile, 2" RELATED [OMIM:618049]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:391614]: should this be "exact" synonym?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -391605,19 +391605,26 @@ is_a: MONDO:0020044 {source="Orphanet:352641"} ! autosomal recessive metabolic c
...
+synonym: "parkinsonism-dystonia, infantile, 2" EXACT [MONDO:0054836]
+synonym: "PKDYS2" RELATED ABBREVIATION [OMIM:618049]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:391616]: should this be "exact" synonym?

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky I have some questions about exact vs related synonyms.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:391614]: yes, fixed

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:391616]: yes, fixed

**Review by nicolevasilevsky — COMMENTED** [review]
updated per your comments, thanks!

**Review by sabrinatoro — APPROVED** [review]
Thank you for updating. It looks good.

**Pull request merged**